torch/_inductor/memory.py

```diff
@@ -6,6 +6,8 @@
 import logging
 from typing import Callable, Dict, List, Set, Tuple, TYPE_CHECKING, TypedDict, Union
 
+from numpy import percentile
```

Comment:
Is it okay to use numpy in inductor?

Reply:
yeah I was somewhat unsure, but I found numpy in requirements.txt so it should be okay: https://github.com/pytorch/pytorch/blob/c25b201583fc28243b87c460a2f18e2531a676e7/requirements.txt#L5